assets/collage.css

```diff
@@ -112,29 +112,29 @@
 
 .collage-card {
   background: rgb(var(--color-background));
-  border-radius: var(--card-corner-radius);
-  border: var(--card-border-width) solid rgba(var(--color-foreground), var(--card-border-opacity));
   height: 100%;
-  padding: var(--card-image-padding);
   position: relative;
+  border-radius: var(--border-radius);
+  border: var(--border-width) solid rgba(var(--color-foreground), var(--border-opacity));
+  padding: var(--image-padding);
 }
 
 .collage-card:after {
-  border-radius: var(--card-corner-radius);
-  box-shadow: var(--card-shadow-horizontal-offset) var(--card-shadow-vertical-offset) var(--card-shadow-blur-radius) rgba(var(--color-shadow), var(--card-shadow-opacity));
   content: '';
   position: absolute;
-  width: calc(var(--card-border-width) * 2 + 100%);
-  height: calc(var(--card-border-width) * 2 + 100%);
-  top: calc(var(--card-border-width) * -1);
-  left: calc(var(--card-border-width) * -1);
   z-index: -1;
+  border-radius: var(--border-radius);
+  box-shadow: var(--shadow-horizontal-offset) var(--shadow-vertical-offset) var(--shadow-blur-radius) rgba(var(--color-shadow), var(--shadow-opacity));
+  width: calc(var(--border-width) * 2 + 100%);
+  height: calc(var(--border-width) * 2 + 100%);
+  top: calc(var(--border-width) * -1);
+  left: calc(var(--border-width) * -1);
 }
 
 .collage-card .media {
```

Comment:
> For the media, it isnt in scope for this PR since this is only affecting cards

Sorry just one last question. Is this documented in a fast follow issue yet? Because the media were originally using card styles, this introduces a regression as I see it.

And forgive me if I missed this somewhere, is the plan to use the global media settings for collage media cards or will they be inheriting from another card type?

Reply:
@duygukalaycioglu Can we make sure we track this in a fast follow for a follow up ticket? 👀 Also wondering what this should be following 🤔 Collection card or product card

Reply:
https://github.com/Shopify/dawn/issues/1822 issue created here